server/services/prismic.js

```diff
@@ -485,7 +485,7 @@ export async function getExhibitionAndEventPromos(query, collectionOpeningTimes)
   // set 'everything' as default time period, when no startDate is provided
   const toDate = !query.startDate ? undefined : query.endDate; ;
   const dateRange = [fromDate, toDate];
-  const allExhibitionsAndEvents = await getAllOfType(['exhibitions', 'events'], {
+  const allExhibitionsAndEvents = await getAllOfType(['exhibitions', 'events', 'installations'], {
```

Comment:
Guess renaming this to include installations starts getting ugly

Reply:
I thought it was maybe overkill, yeah